mediator/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "mediator"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.75"
+aries-vcx = { path = "../aries_vcx" }
+aries_vcx_core = { path = "../aries_vcx_core", features = ["vdrtools_wallet"] }
+async-trait = "0.1.73"
+axum = "0.6"
+axum-macros = "0.3.8"
+cursive = { version = "0.20.0", features = ["crossterm-backend"] }
```

Comment:
This is fairly heavy required dependency which is not needed for mediator itself
```
> cargo tree

├── cursive v0.20.0
│   ├── ahash v0.8.3 (*)
│   ├── cfg-if v1.0.0
│   ├── crossbeam-channel v0.5.8
│   │   ├── cfg-if v1.0.0
│   │   └── crossbeam-utils v0.8.16 (*)
│   ├── crossterm v0.25.0
│   │   ├── bitflags v1.3.2
│   │   ├── libc v0.2.149
│   │   ├── mio v0.8.8 (*)
│   │   ├── parking_lot v0.12.1 (*)
│   │   ├── signal-hook v0.3.17
│   │   │   ├── libc v0.2.149
│   │   │   └── signal-hook-registry v1.4.1
│   │   │       └── libc v0.2.149
│   │   └── signal-hook-mio v0.2.3
│   │       ├── libc v0.2.149
│   │       ├── mio v0.8.8 (*)
│   │       └── signal-hook v0.3.17 (*)
│   ├── cursive_core v0.3.7
│   │   ├── ahash v0.8.3 (*)
│   │   ├── crossbeam-channel v0.5.8 (*)
│   │   ├── enum-map v2.6.3
│   │   │   └── enum-map-derive v0.14.0 (proc-macro)
│   │   │       ├── proc-macro2 v1.0.69 (*)
│   │   │       ├── quote v1.0.33 (*)
│   │   │       └── syn v2.0.38 (*)
│   │   ├── enumset v1.1.3
│   │   │   └── enumset_derive v0.8.1 (proc-macro)
│   │   │       ├── darling v0.20.3 (*)
│   │   │       ├── proc-macro2 v1.0.69 (*)
│   │   │       ├── quote v1.0.33 (*)
│   │   │       └── syn v2.0.38 (*)
│   │   ├── lazy_static v1.4.0 (*)
│   │   ├── log v0.4.20 (*)
│   │   ├── num v0.4.1
│   │   │   ├── num-complex v0.4.4
│   │   │   │   └── num-traits v0.2.17 (*)
│   │   │   ├── num-integer v0.1.45 (*)
│   │   │   ├── num-iter v0.1.43
│   │   │   │   ├── num-integer v0.1.45 (*)
│   │   │   │   └── num-traits v0.2.17 (*)
│   │   │   │   [build-dependencies]
│   │   │   │   └── autocfg v1.1.0
│   │   │   ├── num-rational v0.4.1
│   │   │   │   ├── num-integer v0.1.45 (*)
│   │   │   │   └── num-traits v0.2.17 (*)
│   │   │   │   [build-dependencies]
│   │   │   │   └── autocfg v1.1.0
│   │   │   └── num-traits v0.2.17 (*)
│   │   ├── owning_ref v0.4.1
│   │   │   └── stable_deref_trait v1.2.0
│   │   ├── time v0.3.20 (*)
│   │   ├── unicode-segmentation v1.10.1
│   │   ├── unicode-width v0.1.11
│   │   └── xi-unicode v0.3.0
│   ├── lazy_static v1.4.0 (*)
│   ├── libc v0.2.149
│   ├── log v0.4.20 (*)
│   ├── maplit v1.0.2
│   ├── ncurses v5.101.0
│   │   └── libc v0.2.149
│   │   [build-dependencies]
│   │   ├── cc v1.0.83 (*)
│   │   └── pkg-config v0.3.27
│   ├── signal-hook v0.3.17 (*)
│   ├── term_size v0.3.2
│   │   └── libc v0.2.149
│   ├── unicode-segmentation v1.10.1
│   └── unicode-width v0.1.11
```

Let's make the `tui` separate project which will not be part of `mediator` project per se. If you want, we can leave this for separate PR with additional refactoring.

Reply:
I have moved this (`tui`) behind a feature flag. It should not pull in the heavy dependency by default now. 
We can still do the separation from mediator code at a later point in time.